### PR TITLE
Update docs for TLS

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: content/docs/introduction/faq.md [hunk 1]
```diff
@@ -222,21 +222,14 @@ second, will only run into precision issues after over 285 years.
...
-authentication solutions in every server component.
-
-If you need TLS or authentication, we recommend putting a reverse proxy in
-front of Prometheus. See, for example [Adding Basic Auth to Prometheus with
-Nginx](https://www.robustperception.io/adding-basic-auth-to-prometheus-with-nginx/).
+TLS and basic authentication is gradually being rolled out to the different
+components. Please follow the different releases and changelogs to know which
+components have already implemented it.
+
+The components currently supporting TLS and authentication are:
```
Comment (brian-brazil) [inline content/docs/introduction/faq.md:229]: I'm not sure it makes sense to try and keep a list like this up to date.

Comment (SuperQ) [reply content/docs/introduction/faq.md:229]: I mostly agree, but if we limit it to official components, we only have 15 things to list here. (13 exporters, Prometheus, Alertmanager).

That's a maintainable list.

File: content/docs/introduction/faq.md [hunk 2]
```diff
@@ -222,21 +222,14 @@ second, will only run into precision issues after over 285 years.
...
+
+- Prometheus 2.24.0 and later
+- Node Exporter 1.0.0 and later
```
Comment (SuperQ) [inline content/docs/introduction/faq.md:232]:
```suggestion
- node\_exporter 1.0.0 and later
```

File: content/docs/introduction/faq.md (hunk 1 above)
Comment (roidelapluie) [reply content/docs/introduction/faq.md:229]: This is not really something to maintain, it is a one-shot.

Comment (roidelapluie) [reply content/docs/introduction/faq.md:229]: on 15 times a one-shot, and we can even batch them when adding them.

**Comment by brian-brazil** [comment]
:+1:

**Review by grobie — APPROVED** [review]

File: content/docs/introduction/faq.md (hunk 2 above)
Comment (roidelapluie) [reply content/docs/introduction/faq.md:232]: We use Node Exporter in the security part, and Prometheus is with a capital lette

**Review by SuperQ — APPROVED** [review]

**Pull request merged**